stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -95,6 +106,20 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
         bottomSheetController.hide()
     }
 
+    private fun updateToolbarButton(isStackEmpty: Boolean) {
+        if (isStackEmpty) {
```

Comment:
You also might be able to simplify this as toolbar.navigationIcon = if()else()

Reply:
How about this since there are two statements for each case?